testsuite/tests/tool-caml-tex/redirections.ml

```diff
@@ -0,0 +1,19 @@
+(* TEST
+   reference="${test_source_directory}/redirections.reference"
+   output="redirections.output"
+   script = "${ocamlrun} ${ocamlsrcdir}/tools/caml-tex -repo-root ${ocamlsrcdir} ${test_source_directory}/${test_file} -o ${output}"
```

Comment:
check-typo complains about that line being too long, can you use `\` to break it after `${ocamlsrcdir}` for example?